documentation/sphinx/source/installation.rst

```diff
@@ -101,6 +103,60 @@ In the context of SCT, it can be used:
 See https://github.com/neuropoly/sct_docker for more information.
 
 
+Using pip (experimental)
+************************
+
+SCT can be installed using pip, with some caveats:
+
+- The installation is done in-place, so the folder containing SCT must
+  be kept around
+
+- In order to ensure coexistence with other packages, the dependency
+  specifications are loosened, and it is possible that your package
+  combination has not been tested with SCT.
+
+  So in case of problem, try again with the reference installation,
+  and report a bug indicating the dependency versions retrieved using
+  `sct_check_dependencies`.
+
+
+Procedure:
+
+#. Retrieve the SCT code to a safe place
+
+   Clone the repository and hop inside:
+
+   .. code:: sh
+
+      git clone https://github.com/neuropoly/spinalcordtoolbox
+
+      cd spinalcordtoolbox
+
+#. Checkout the revision of interest, if different from `master`:
+
+   .. code:: sh
+
+      git checkout ${revision_of_interest}
+
+#. If numpy is not already on the system, install it, either using
```

Comment:
there is probably a good reason for asking to install numpy separately, just curious why?
also, don't we want to specify a specificy numpy version?

Reply:
There was one package (I forgot which one) that didn't indicate a build-time dependency on numpy but actually required it.

Reply:
Actually the doc is obsolete because I have removed the step of installing numpy from the travis file. The deps should be OK as of today.

Reply:
So the step is unnecessary, but doesn't hurt since the end result is the same. I'd fix it later.